new-colorlib/style.css

```diff
@@ -0,0 +1,156 @@
+* {
+  margin: 0;
+}
+/*HEADER*/
+.header-container {
+  background: rgb(255, 0, 0);
+  height: 150px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+.header-left {
+  color: #f5efef;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.header-img {
+  height: 40px;
+  border-radius: 20px;
+  margin-right: 10px;
+}
+.video-mag {
+  font-size: 50px;
+  font-weight: 600;
+}
+.header-right {
+  background: url(../new-colorlib/img/abstract-digital-grid-black-background.jpg);
+  height: 90px;
+  width: 50%;
+  color: #f5efef;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-size: cover;
+}
+/*NAVBAR*/
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 100px;
+  background-color: black;
+  color: #ccc;
+  padding-right: 50px;
+  font-size: 20px;
+  font-weight: 500;
+  margin-bottom: 10px;
+}
+.nav-list ul li {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+}
+ul li {
+  height: 100px;
+  padding-left: 20px;
+  padding-right: 20px;
+  display: flex;
+}
+ul li:hover {
+  background-color: rgb(233, 29, 73);
+  transition: 0.5s;
+}
+.search-img {
+  height: 20px;
+}
+input {
+  height: 30px;
+  width: 300px;
+  background-color: #888686;
+  border: none;
+  border-radius: 3px;
+  padding-left: 20px;
+}
+.input-nav {
+  position: relative;
+}
+.search-img {
+  position: absolute;
+  right: 5px;
+  top: 5px;
+}
+.home {
+  background-color: rgb(233, 29, 73);
+}
+
+/*Body Section*/
+
+.grid-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  height: 800px;
+  gap: 10px;
+  padding-left: 12px;
+  padding-right: 12px;
+}
+.grid-left {
+  background-image: url(../new-colorlib/img/1.jpg);
+  background-size: cover;
+  color: white;
+  position: relative;
+  -webkit-backdrop-filter: blur(8px);
+}
+.grid-right {
+  display: grid;
+  gap: 10px;
+}
+.grid-sm1 {
+  background-image: url(../new-colorlib/img/2.jpg);
```

Comment:
- [ ] Link to this image
[grid top right image](https://preview.colorlib.com/theme/videomag/img/bg-img/2.jpg.webp)